waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -861,8 +887,16 @@ proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf) {.raises: [Defect, Value
       let memKeyPair = keyPair.toMembershipKeyPairs()[0]
       # mount the rln relay protocol in the on-chain/dynamic mode
       waitFor node.mountRlnRelayDynamic(memContractAddr = ethMemContractAddress, ethClientAddr = ethClientAddr, memKeyPair = some(memKeyPair), memIndex = some(rlnRelayIndex), ethAccAddr = ethAccountAddr, ethAccountPrivKey = ethAccountPrivKey, pubsubTopic = conf.rlnRelayPubsubTopic, contentTopic = conf.rlnRelayContentTopic)
+    elif fileExists(RLN_CREDENTIALS_FILEPATH):
+      var credentials = readPersistentRlnCredentials()
```

Comment:
For more readability and ease of debugging, I suggest passing the rln credential path as an argument to this proc `readPersistentRlnCredentials`.

Reply:
Suggestion Incorporated.

Reply:
Refer 3bbd64b6650e53a580919af51b5b48478f5ad2e9